### PR TITLE
Members with no team filtered

### PR DESCRIPTION
**Pull request by ssjananna**
Hey take a look. Ran test and coverage was 100%.

### CODE REVIEW

File: app/js/common.js [hunk 1]
```diff
@@ -11,3 +11,17 @@ function isAvailable(team) {
...
+
+/**
+ * filter members that do not have a team yet
+ *
+ * @param array of members
+ * @return filtered members
+ */
+
+function hasNoTeam(member){
+	return team===undefined;
```
Comment (chautsunman) [inline app/js/common.js]: `return member.team === undefined;`

File: app/unit_tests/common_test.js [hunk 2]
```diff
@@ -28,3 +28,34 @@ describe('Test common_test.js', function() {
         });
     });
 });
+
+describe('Test common_test.js', function() {
+    // A test case of MembersWithNoTeam
+    describe('MembersWithNoTeam Coverage Test', function() {
```
Comment (chautsunman) [inline app/unit_tests/common_test.js:34]: write another unit test that some members do not have the `team` property

File: app/js/common.js [hunk 3]
```diff
@@ -11,3 +11,17 @@ function isAvailable(team) {
 function getAvailableTeam(teams) {
 	return teams.filter(isAvailable);
 }
+
+/**
```
Comment (chautsunman) [inline app/js/common.js:15]: move this documentation comment to just before `function hasNoTeam(member)`

File: app/js/common.js [hunk 4]
```diff
@@ -11,3 +11,17 @@ function isAvailable(team) {
...
+ * filter members that do not have a team yet
+ *
+ * @param array of members
```
Comment (chautsunman) [inline app/js/common.js]: `@param members array of members`

File: app/js/common.js (hunk 3 above)
Comment (ssjananna) [reply app/js/common.js:15]: it already is right above hasNoteam(member)

**Pull request merged**